.gitignore

```diff
@@ -1,8 +1,6 @@
-/.*
-!/.gitignore
-!/.jscsrc
-!/.jshintrc
-!/.travis.yml
-/bower_components/
-/node_modules/
-/output/
```

Comment:
I copied it from another project. This package doesn't use anything from NPM, but maybe I should keep the dotfiles ..